bindings/java/src/main/native/jucx_common_def.cc

```diff
@@ -184,6 +184,11 @@ void send_callback(void *request, ucs_status_t status)
     ucp_request_free(request);
 }
 
+void recv_callback(void *request, ucs_status_t status, ucp_tag_recv_info_t *info)
+{
+    send_callback(request, status);
```

Comment:
need to rename send_callback to 'ucp_request_callback' or similar, it's weird that recv_callback calls send_callback